src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -449,6 +473,34 @@ public List<String> retrieveUserDatasetRoles(TestConfiguration.User user, UUID d
     return validateResponse(response, "retrieving dataset roles", HttpStatus.OK, null);
   }
 
+  // getting a resource's policies
+  public DataRepoResponse<PolicyResponse> retrievePoliciesRaw(
+      TestConfiguration.User user, UUID resourceId, IamResourceType iamResourceType)
+      throws Exception {
+    String pathPrefix;
+    switch (iamResourceType) {
```

Comment:
Should we also add a case for the billing profile for completeness?

Reply:
Good call, I will add this!